src/components/MainLayout/FooterView.vue

```diff
@@ -2,8 +2,8 @@
   <div class="hr" />
   <footer class="footer">
     <div class="row items-center justify-center q-gutter-md">
-      <a href="https://twitter.com/ASE_lab_" target="_blank" class="icon">
-        <q-btn icon="mdi-twitter" flat round />
+      <a href="https://x.com/ASE_lab_" target="_blank" class="icon-x">
+        <q-btn icon="𝕏" flat round class="q-btn__content" />
```

Comment:
多分これで行ける気が
ローカルで試してもらえますか？
```suggestion
        <q-btn icon="𝕏" flat round />
```